cmd/config.go

```diff
@@ -31,6 +31,9 @@ func configFlagSet() *pflag.FlagSet {
 	flags.StringArrayP("out", "o", []string{}, "`uri` for an external metrics database")
 	flags.BoolP("linger", "l", false, "keep the API server alive past test end")
 	flags.Bool("no-usage-report", false, "don't send anonymous stats to the developers")
+	flags.Bool("web-dashboard", false, "enable web dashboard")
+	// we don't want to introduce it, in future releases logic may change
+	flags.MarkHidden("web-dashboard") //nolint:errcheck,gosec
```

Comment:
```suggestion
```
What's the point here? If we don't want to introduce it then postpone the addition in a dedicated PR in the future.

Reply:
Done, removed.